app/controllers/restful_redirect_controller.rb

```diff
@@ -8,9 +8,11 @@ def index
       if record
         if record.kind_of?(EmsConfiguration)
           redirect_to(:controller => 'ems_configuration', :action => 'show', :id => params[:id])
-        elsif %w[ManageIQ::Providers::AnsibleTower::AutomationManager].include?(record.type)
+        elsif record.kind_of?(ManageIQ::Providers::ExternalAutomationManager)
           redirect_to(:controller => 'ems_automation', :action => 'show', :id => params[:id])
-        elsif %w[ManageIQ::Providers::EmbeddedAnsible::AutomationManager].include?(record.type)
+        elsif record.kind_of?(ManageIQ::Providers::AnsibleTower::AutomationManager)
+          redirect_to(:controller => 'ems_automation', :action => 'show', :id => params[:id])
```

Comment:
This two lines can be removed now.  `ManageIQ::Providers::AnsibleTower::AutomationManager` is a kind of `ManageIQ::Providers::ExternalAutomationManager`, so it will always hit the prior elsif statement and never hit this one.

Reply:
@kavyanekkalapu I think this comment hasn't been addressed.

Reply:
Or I think you deleted the wrong else branch?

Reply:
Good catch :) .. let me delete the correct one.. there are many similar ones and got confused..